torchvision/models/quantization/googlenet.py

```diff
@@ -141,18 +141,34 @@ def googlenet(
     quantize: bool = False,
     **kwargs: Any,
 ) -> QuantizableGoogLeNet:
-    r"""GoogLeNet (Inception v1) model architecture from
-    `"Going Deeper with Convolutions" <http://arxiv.org/abs/1409.4842>`_.
+    """GoogLeNet (Inception v1) model architecture from `Going Deeper with Convolutions <http://arxiv.org/abs/1409.4842>`__.
 
-    Note that quantize = True returns a quantized model with 8 bit
+    Note that ``quantize = True`` returns a quantized model with 8 bit
     weights. Quantized models only support inference and run on CPUs.
     GPU inference is not yet supported
 
+    The required minimum input size of the model is 15x15.
+
     Args:
-        weights (GoogLeNet_QuantizedWeights or GoogLeNet_Weights, optional): The pretrained
-            weights for the model
-        progress (bool): If True, displays a progress bar of the download to stderr
-        quantize (bool): If True, return a quantized version of the model
+        weights (:class:`~torchvision.models.quantization.GoogLeNet_QuantizedWeights` or :class:`~torchvision.models.GoogLeNet_Weights`, optional): The
+            pretrained weights for the model. See
+            :class:`~torchvision.models.quantization.GoogLeNet_QuantizedWeights` below for
+            more details, and possible values. By default, no pre-trained
+            weights are used.
+        progress (bool, optional): If True, displays a progress bar of the
+            download to stderr. Default is True.
+        quantize (bool, optional): If True, return a quantized version of the model. Default is False.
+        **kwargs: parameters passed to the ``torchvision.models.quantization.QuantizableGoogLeNet``
+            base class. Please refer to the `source code
+            <https://github.com/pytorch/vision/blob/main/torchvision/models/quantization.googlenet.py>`_
+            for more details about this class.
+
+    .. autoclass:: torchvision.models.quantization.GoogLeNet_QuantizedWeights
+        :members:
+
+    .. autoclass:: torchvision.models.GoogLeNet_Weights
+        :members:
+        :noindex:
```

Comment:
Question: Why noindex here?

Reply:
if we see error logs when we build docs we actually need this in many places. Any way to inject these?

Reply:
I think this has to be done manually. But there should be very few cases where this is going to be needed. It's the only use-case we have for `:noindex:` so far.

(Note that we get tons of warnings in the docs right now but it's normal, they'll be all gone once we remove `models.rst`).